src/web/pages.rkt

```diff
@@ -96,13 +101,21 @@
     (for/list ([point points])
       (for/list ([value point]) 
         (real->ordinal value repr))))
-
+        
   (define vars (test-vars test))
   (define bits (representation-total-bits repr))
   (define start-error (map ulps->bits-tenths start-errors))
-  (define target-error (and target-errors (map ulps->bits-tenths target-errors)))
+  (define target-error (and target-errors (map (lambda (alt-error) (map ulps->bits-tenths alt-error)) target-errors)))
```

Comment:
If you allowed empty lists, you could drop the or.

Reply:
I think you're referring to "and" but I refactored this